shared/ui-components/src/prototypes/AppCard.stories.tsx

```diff
@@ -12,7 +12,7 @@ export const AppCard = () => (
     <div className="app-card__header border-bottom">
       <div className="app-card__back">
         <a href="#" className="lined">
-          Back
+          Whack
```

Comment:
Doesn't really matter since it's in a prototype story, but was this really intended?